### ISSUE TITLE
.perlcriticrc config file is now being ignored

### ISSUE THREAD

**Issue by Qarj** [github/super-linter#2784]
## Describe the bug

Since this pull request https://github.com/github/super-linter/pull/2684/commits/cdbbce5ff009e1e18bdeec0c50d7b1a049c6da15 was merged, I am unable to specify my perl critic profile file or disable the unwanted community theme.

## Expected behavior

.perlcriticrc file should be respected

## Steps to Reproduce

Navigate to https://github.com/Qarj/WebImblaze/runs/5981458220?check_suite_focus=true and see that even though I disable Community::EmptyReturn in the perl critic config https://github.com/Qarj/WebImblaze/blob/lintingFixes/.perlcriticrc file it is being ignored.

## Logs

Snippet from linting logs showing that linting errors are being reported on an ignored lint

```txt
/github/workspace/wi.pl:2222:5 [1 Community::EmptyReturn] return called with no arguments near 'return;'
/github/workspace/wi.pl:2794:5 [1 Community::EmptyReturn] return called with no arguments near 'return;'
/github/workspace/wi.pl:2996:5 [1 Community::EmptyReturn] return called with no arguments near 'return;'
/github/workspace/wi.pl:3008:9 [1 Community::EmptyReturn] return called with no arguments near 'return;'
```

## Additional context

None.
